### ISSUE TITLE
Idea: progress bar for file upload

### ISSUE THREAD

**Issue by wwhtrbbtt** [Apathetic-Development-Team/spore#10]
Hi, I think a small progress bar for uploads with an estimated time would be extremely nice.

**Comment by Forsol** [comment]
Yes this was the plan after we got threading down, will think up some ideas for implementation.

**Comment by wwhtrbbtt** [comment]
Nice! Ill work on the GUI a bit and implement some things that Omni requested

**Comment by Forsol** [comment]
Should the progress bar reflect individual uploads or the overall process of uploading (50% after archiving, 100 when all is finished)

**Comment by wwhtrbbtt** [comment]
I think individual uploads would look better.

**Comment by CYBERLUST** [comment]
Current solution can be implemented with our existing libs.
